app/jobs/get_suggestions_job.rb

```diff
@@ -0,0 +1,35 @@
+class GetSuggestionsJob < ApplicationJob
+  queue_as :default
+
+  def perform(job_status)
+    @job_id = job_status.jobId
+
+    SpotifyTrackGetter.send_spotify_request
+
+    if SpotifyTrackGetter.get_spotify_tracks
```

Comment:
There's some state being tracked here -- `.send_spotify_request` must be called before `.get_spotify_tracks` before `.track_data`. How about extracting an object to store this state on an instance, to avoid mutating global state?